common/models/wordbreakers/build.sh

```diff
@@ -26,42 +26,31 @@ builder_describe "Builds the predictive-text wordbreaker implementation module"
 
 builder_describe_outputs \
   configure          /node_modules \
-  build              build/lib/index.mjs
+  build              build/obj/index.js
 
 builder_parse "$@"
 
-### CONFIGURE ACTIONS
+builder_run_action configure verify_npm_setup
+builder_run_action clean rm -rf build/
 
-if builder_start_action configure; then
-  verify_npm_setup
-  builder_finish_action success configure
-fi
-
-### CLEAN ACTIONS
-
-if builder_start_action clean; then
-  rm -rf build/
-  builder_finish_action success clean
-fi
 
 ### BUILD ACTIONS
 
-if builder_start_action build; then
+do_build ( ) {
   tsc -b
-  node build-bundler.js
 
   # Declaration bundling.
   tsc --emitDeclarationOnly --outFile ./build/lib/index.d.ts
+}
 
-  builder_finish_action success build
-fi
+builder_run_action build do_build
 
-if builder_start_action test; then
+do_test ( ) {
   if builder_has_option --ci; then
     c8 mocha -reporter mocha-teamcity-reporter
   else
     c8 mocha
   fi
+}
 
-  builder_finish_action success test
-fi
\ No newline at end of file
+builder_run_action test do_test
```

Comment:
In this file also, suggest grouping all functions together above the `builder_run_action` statements, and table-formatting the `builder_run_action` statements